examples/networking/zigbee-lightbulb/main.c

```diff
@@ -0,0 +1,176 @@
+
+int cmd_zconfig(int argc, char *argv[]);
+SHELL_COMMAND(zconfig, "ZBOSS configuration", cmd_zconfig);
+
+SHELL_COMMAND(reset, "reset the device and perform reboot", zb_reset);
+
+void set_led(zb_uint8_t param)
+{
+
+#ifdef WS281X_PARAM_NUMOF
+    ws281x_prepare_transmission(&led_dev);
+    if (param) {
+        ws281x_set(&led_dev, 0, led_white);
+    }
+    else {
+        ws281x_set(&led_dev, 0, led_coff);
+    }
+    ws281x_write(&led_dev);
+    ws281x_end_transmission(&led_dev);
+#elif defined LED0_ON
+    if (param) {
+        LED0_ON;
+    }
+    else {
+        LED0_OFF;
+    }
+#endif
+}
+
+/* Cluster setup */
+static zb_zcl_groups_srv_attr_t group_attrs = { .name_support = 1 };
+
+static zb_zcl_on_off_srv_attr_t on_off_attrs = {
+    .on_off = ZB_FALSE,
+    .global_scene_ctrl = ZB_TRUE,
+    .on_time = 0x0000,
+    .off_wait_time = 0x0000,
+    .set_state = set_led,
+};
+static zb_zcl_basic_srv_attr_t basic_attrs;
+
+int main(void)
+{
+    /* setup ws281x led */
+#ifdef WS281X_PARAM_NUMOF
+    ws281x_params_t led_params;
+    led_params.numof = WS281X_PARAM_NUMOF;
+    led_params.pin = WS281X_PARAM_PIN;
+    ws281x_init(&led_dev, &led_params);
+#endif
+    /* we need a message queue for the thread running the shell in order to
+     * receive potentially fast incoming networking packets */
+    msg_init_queue(_main_msg_queue, MAIN_QUEUE_SIZE);
+    puts("RIOT Zigbee ZLL example application");
+
+    /* set device descriptor */
+    ZB_DECLARE_SIMPLE_DESC(3/* in clusters. Groups, basic, on-off */, 1/* out clusters. on-off */);
+    static zb_af_simple_desc_3_1_t desc;
+    zb_set_simple_descriptor((zb_af_simple_desc_1_1_t *)&desc,
+                              11, /* endpoint */
+                              ZB_HA_PROFILE_ID,
+                              ZB_HA_ON_OFF_LIGHT_DEV_ID,
+                              1, /* app version */
+                              3, /* input_cl count*/
+                              1 /* output_cl count*/
+                            );
+    zb_set_input_cluster_id((zb_af_simple_desc_1_1_t *)&desc, 0, ZB_BASIC_CLUSTER_ID);
+    zb_set_input_cluster_id((zb_af_simple_desc_1_1_t *)&desc, 1, ZB_GROUPS_CLUSTER_ID);
+    zb_set_input_cluster_id((zb_af_simple_desc_1_1_t *)&desc, 2, ZB_ON_OFF_CLUSTER_ID);
+
+    zb_set_output_cluster_id((zb_af_simple_desc_1_1_t *)&desc, 0, ZB_ON_OFF_CLUSTER_ID);
+
+    zb_add_simple_descriptor((zb_af_simple_desc_1_1_t *)&desc);
+
+    /* set cluster attributes */
+    zb_zcl_basic_srv_set_defaults(&basic_attrs);
+
+    basic_attrs.app_version = 1;
+    basic_attrs.power_source = ZB_ZCL_BASIC_POWER_SRC_MAINS;
+    basic_attrs.phys_environment = ZB_ZCL_BASIC_PHYS_ENV_OFFICE;
+    basic_attrs.generic_device_type = ZB_ZCL_BASIC_GEN_DEV_TYPE_LED_BULB;
+
+    char manufacturer[] = "RIOT OS";
+    basic_attrs.manufacturer_name_len = sizeof(manufacturer);
+    ZB_MEMCPY(basic_attrs.manufacturer_name, manufacturer, sizeof(manufacturer));
+
+    char model_id[] = "nRF52840DK";
```

Comment:
```suggestion
    char model_id[] = RIOT_BOARD;
```

The variable `RIOT_BOARD` specifies the board name used for the compilation. Then it's not hardcoded.